.gitignore

```diff
@@ -1,7 +1,7 @@
 # Uncompressed project data files
-edgePy/data/*.csv
-edgePy/data/*.tsv
-edgePy/data/*.txt
+# edgePy/data/*.csv
+# edgePy/data/*.tsv
+# edgePy/data/*.txt
```

Comment:
Can you comment these back out and only provide gzipped reference data?

Reply:
yes, I forgot to uncomment them, - they're not part of the process of copying the data.

However, for the reference data, I intentionally made the data files (.txt) non-gzipped so that others could recreate them easily.  They're meant to be examples, for others to copy.  The npz file is already gzipped.

Future data sets should all be gzipped.